### PR TITLE
Fix use with DllPlugin

### PR DESCRIPTION
**Pull request by mzgoddard**
Related to #18

DllPlugin needs a method on modules to work called `libIdent` to build
an identier that DllReferencePlugin uses to see if it should use the
dll contained module instead of letting a normal module be built.

### CODE REVIEW

**Comment by mzgoddard** [comment]
Depending on the details of the case @MoOx is exploring in #18. This may fix it. Otherwise it fixes building a Dll with HardSource. I don't believe it has an effect on the performance of using the dll with DllReferencePlugin and HardSource.

File: index.js [hunk 1]
```diff
@@ -557,6 +557,7 @@ HardSourceWebpackPlugin.prototype.apply = function(compiler) {
     }
 
     compilation.modules.forEach(function(module, cb) {
+      console.log(module.request, module.cacheable, (module instanceof HardModule), (module instanceof NormalModule));
```
Comment (MoOx) [inline index.js]: Maybe this console is not necessary?

**Comment by MoOx** [comment]
Thanks for your work. Hope this will make things better :)

File: index.js (hunk 1 above)
Comment (mzgoddard) [inline index.js]: Indeed.

Comment (mzgoddard) [inline index.js]: Thanks for the review.

**Pull request merged**